chucks-pick3-mobile/src/app/components/pick3-draw-date-info-section/pick3-draw-date-info-section.ts

```diff
@@ -229,8 +236,12 @@ export class Pick3DrawDateInfoSection implements OnInit {
                     this.setDrawState(this.data, Pick3DrawTimeCardStateEnum.Pick3DrawTimeCardStateEnum.NOT_DRAWN_YET);
             }
         }
-
-        this.showCountDownToDrawing = false;
+        const path = location.pathname;
```

Comment:
Combine duplicate code into single method and reuse the method here.

Reply:
Reduced to single method and reused.